code/modules/clothing/suits/hood.dm

```diff
@@ -81,6 +85,12 @@
 			return
 		RemoveHood()
 
+/obj/item/clothing/suit/hooded/proc/try_to_deploy()
+	return TRUE
+
+/obj/item/clothing/suit/hooded/proc/on_hood_deploy()
+	return
```

Comment:
Document these procs, particularly try_to_deploy() and what its return type does

Reply:
I need to remove these, diffrent solution used, thanks for catching!

Reply:
Removed!